REPORTING_ISSUES.md

```diff
@@ -0,0 +1,171 @@
+# REPORTING ISSUES FOR CUPS PROJECT
+
+> Disclaimer: We provide only best effort support for CUPS releases older than the latest release. This means we try to reproduce the issue on the latest release if it is possible, and if we can't reproduce, we can give a hint what can be the problem and ask you to contact your distribution support.
+
+The steps you are asked to do differ based on how your printer is connected (USB/network) and whether you use driverless printing or not. Since the connection is obvious for the user, the paragraph below will tell you how to find out whether driverless printing is used.
+
+
+## HOW TO FIND OUT THAT I USE DRIVERLESS PRINTING
+
+For network printers:
+* your printer is seen by `lpstat -e`, but not by `lpstat -a` - this means you print via temporary queue, which works only via driverless means,
+
+or
+
+* your printer is seen by `lpstat -a`, its PPD file at `/etc/cups/ppd` has `IPP Everywhere` or `driverless` in its `Nickname` entry and its connection is `ipp` or `ipps` when you check the connection uri (f.e. by `lpstat -v <printer_name>`)
+
+If any of them apply, your device work in driverless mode.
+
+For USB printers:
+* your printer is listed if you enter:
+
+```
+    $ sudo ipp-usb check
+```
+
+in case the printer is not listed or the command is not found, your device doesn't work in driverless mode.
+
+
+## HOW TO REPORT THE ISSUE TO CUPS PROJECT
+
+Please do the steps below and provide the mentioned information (**use attachments for blocks of text longer than 10 lines** - put the text into a file, rename it to have .txt suffix and click on `pasting them` string under comment box in the issue) in your initial comment if you file an issue:
+
+* tell us what is **your OS**
+* tell us your **CUPS version**
+* in case you compile CUPS by yourself, tell us **all configuration options** you pass into `./configure`
+* **describe** the problem
+* mention you **printer model**
```

Comment:
Typo:
```
mention you **printer model**
```
should be
```
mention your **printer model**
```
FYI: cf.
https://en.opensuse.org/SDB:How_to_Report_a_Printing_Issue#In_any_case_without_exception_tell_the_exact_printer_model_name

Reply:
Typo fixed, thank you for the review and the link! Of course I've forgot about some more info which can be provided, although I wrote it into our Fedora docs several years ago :D .
I didn't want to go too deep with debugging (which is great the link!) in the doc, but focus more on data and don't bore the reporter too much (which I failed, obviously :D )